pwm2rpm: "[[15000.0, 1], [15200.0, 2],[15250.0, 3], [15257.0, 4], [15260.0, 200], [15265.0, 220], [15270.0, 245], [15275.0, 255], [15280.0, 295], [15290.0, 315], [15300.0, 340], [15310.0, 358], [15320.0, 415]]"
pwm2angle: "[[11450.0, -25.0], [11765.0, -23.0], [12080.0, -21.0], [12395.0, -19.0], [12710.0, -17.0], [13025.0, -15.0], [13500.0, -13.0], [13820.0, -11.0], [14145.0, -9.0], [14470.0, -7.0], [14790.0, -5.0], [15115.0, -3.0], [15440.0, -1.0], [15600.0, 0.0], [15650.0, 0.0], [15925.0, 2.0], [16120.0, 4.0], [16470.0, 6.0], [16740.0, 8.0], [17015.0, 10.0], [17290.0, 12.0], [17425.0, 13.0], [17705.0, 15.0], [17985.0, 17.0], [18265.0, 19.0], [18550.0, 21.0], [18830.0, 23.0], [19110.0, 25], [19250.0, 26.0]]"
angle_size: 0.2
wheel_radius: 0.08
